### PR TITLE
Wording Changes

### PR DESCRIPTION
**Pull request by XavierSchiller**
A small example on wording the introduction(Please do edit what I've left in brackets as needed.)

### CODE REVIEW

File: src/pages/basics/introduction.js [hunk 1]
```diff
@@ -11,42 +11,40 @@ const IntroductionPage = () => (
...
-      color information while applying any lighting and darkening effects on
-      that image.
+      color information while applying any lighting effects on
+      said image.
     </p>
     <p>
-      While shaders are still used for this purpose, they can now also be used
-      to perform other types of graphical and compute tasks.
+      While shaders are mainly used for this purpose, they can now also be used
+      to perform other types of computational tasks.
```
Comment (sabarnac) [inline src/pages/basics/introduction.js]: compute tasks instead of computational?

Comment (XavierSchiller) [reply src/pages/basics/introduction.js]: As mentioned, do take care when trying to add detail and remove detail. Sometimes, details don't matter, and here, it certainly doesn't.

File: src/pages/basics/introduction.js [hunk 2]
```diff
@@ -11,42 +11,40 @@ const IntroductionPage = () => (
...
-      set of data. This was later replaced with the current shading system used
-      by most GPUs today.
+      Initally, graphics cards (GPUs) didn't support programmable shaders. Initially, GPUs
+      would have a fixed-function pipeline(where one would call specific units),
+      which could only perform specific actions. 
+    </p>
+    <p>  
+      This was later replaced with the current shading system used today.
     </p>
     <h3>What are the different types of shaders?</h3>
```
Comment (sabarnac) [inline src/pages/basics/introduction.js]: This section is removed, so not needed.

Comment (XavierSchiller) [reply src/pages/basics/introduction.js]: Will edit it out.

File: src/pages/basics/introduction.js [hunk 3]
```diff
@@ -92,13 +90,12 @@ const IntroductionPage = () => (
...
-      , where every shader type is practically the same in the functions and
-      APIs they have access to. This makes them simpler to implement in hardware
-      and easy to program in software as well.
+      , where every shader type is practically the same. 
+      This makes them simpler to implement in hardware and easy to program in software as well.
     </p>
     <h3>What do I need to know before getting started?</h3>
     <p>
-      A beginner understanding of matrix and vector math will be helpful when
+      (Make this a list)A beginner understanding of matrix and vector math will be helpful when
```
Comment (sabarnac) [inline src/pages/basics/introduction.js]: How about

Go through {link} of {link} to learn about the necessary details vector and matrix math, even if you have an understanding of it. This information is vital as it provides a base for understanding all the math that'll be used later on.

As a side note...

Comment (sabarnac) [reply src/pages/basics/introduction.js]: Since there isn't much of a pre-requisite, a list shouldn't be necessary

Comment (XavierSchiller) [reply src/pages/basics/introduction.js]: That would do, yes.

**Pull request merged**